extensions/sql-database-projects/src/models/project.ts

```diff
@@ -170,6 +170,13 @@ export class Project implements ISqlProject {
 						} else {
 							// only add file if it wasn't already added
 							if (!this._files.find(f => f.relativePath === relativePath)) {
+								// SSDT encodes @ to %40, so we need to decode that so that it shows correctly in the tree
+								// I didn't find any other characters that get encoded, so this check is specifically for %40 to have lower risk of accidentally breaking something
+								// but we can decode all relative paths in the future if we find other characters that get encoded
+								if (relativePath.includes('%40')) {
+									relativePath = decodeURIComponent(relativePath);
```

Comment:
I originally had this decode all relative paths, but since this bug has only been reported for @ as %40 and I didn't find any other characters that get encoded, I'm only decoding for that case for now so there's a lower risk something else might accidentally break